### PR TITLE
de.json language pack

### PR DESCRIPTION
**Pull request by Ardenian**
Needs #228 to be solved and closed

### CODE REVIEW

File: src/languages/de.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        "enableFavRequest": "Aktiviere automatische Nachfrage f�r Raids f�r Favoriten?",
+        "favIdToJoin": "Favorisierte Ids zum beitreten ( mehrere mit Komma trennen)",
+        "worldserver": "Worldserver?"
+    },
+    "ui": {
+        "settingInfo": "Dieses Skript wird immer alle Raids importieren, die Sie manuell auf der Website festgelegt haben und, falls aktiviert, wird es auch alle Favouriten aus Suchergebnissen importieren.",
+        "maxPopupsFrame": "Maximale Popups/Frames beim Beitreten von Raids",
+        "timeAutoJoin": "Zeit, um langsam automatisch Raids beizutreten ( l�dt Spiel mehrere Male neu). Format ist [Hours]:[Minutes] ohne anf�hrende Nullen, also 7:1 ist gut, 07:01 ist schlecht",
+        "getFavFrom": "Bekomme Suchfavoriten aus ",
+        "raidsearch": "Idrinth's Raidsearch",
```
Comment (Idrinth) [inline src/languages/de.json]: 's is english only

File: src/languages/de.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        "joiningDuration": "Sekunden, die das Spiel zum Laden braucht, bevor man Raids beitreten kann",
+        "enableChat": "Chat aktivieren ( ben�tigt Neustart des Skripts)",
+        "enableFavRequest": "Aktiviere automatische Nachfrage f�r Raids f�r Favoriten?",
+        "favIdToJoin": "Favorisierte Ids zum beitreten ( mehrere mit Komma trennen)",
+        "worldserver": "Worldserver?"
+    },
+    "ui": {
+        "settingInfo": "Dieses Skript wird immer alle Raids importieren, die Sie manuell auf der Website festgelegt haben und, falls aktiviert, wird es auch alle Favouriten aus Suchergebnissen importieren.",
+        "maxPopupsFrame": "Maximale Popups/Frames beim Beitreten von Raids",
+        "timeAutoJoin": "Zeit, um langsam automatisch Raids beizutreten ( l�dt Spiel mehrere Male neu). Format ist [Hours]:[Minutes] ohne anf�hrende Nullen, also 7:1 ist gut, 07:01 ist schlecht",
```
Comment (Idrinth) [inline src/languages/de.json]: wasn't completely translated

File: src/languages/de.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        "disableSpecific": "Deaktiviere automatisches Beitreten f�r bestimmte Raids"
+    },
+    "button": {
+        "ok": "Ok",
+        "cancel": "Schlie�en",
+        "reloadGame": "Game neuladen",
+        "reloadScript": "Skript neuladen",
+        "refreshFBGameLogin": "Erneuere Facebook Login",
+        "ngRaidJoin": "NG Raid Beitritt ( langsam!)",
+        "disableTimedAutoJoin": "Deaktiviere zeitlich-automatisches Beitreten",
```
Comment (Idrinth) [inline src/languages/de.json]: zeitlich-automatisch? What about something like geplantes?

File: src/languages/de.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+            "restart": "Starte das Beitreten von Raids neu"
+        },
+        "disableSpecific": "Deaktiviere automatisches Beitreten f�r bestimmte Raids"
+    },
+    "button": {
+        "ok": "Ok",
+        "cancel": "Schlie�en",
+        "reloadGame": "Game neuladen",
+        "reloadScript": "Skript neuladen",
+        "refreshFBGameLogin": "Erneuere Facebook Login",
```
Comment (Idrinth) [inline src/languages/de.json]: Game is missing, it's not about logging ino facebook.

File: src/languages/de.json [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+            "favs": "Importiere Favoriten manuell",
+            "restart": "Starte das Beitreten von Raids neu"
+        },
+        "disableSpecific": "Deaktiviere automatisches Beitreten f�r bestimmte Raids"
+    },
+    "button": {
+        "ok": "Ok",
+        "cancel": "Schlie�en",
+        "reloadGame": "Game neuladen",
+        "reloadScript": "Skript neuladen",
```
Comment (Idrinth) [inline src/languages/de.json]: this and the above sound weird, "Lade X neu" maybe?

File: src/languages/de.json [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+        "imports": {
+            "manually": "Importiere alle manuell",
+            "favs": "Importiere Favoriten manuell",
+            "restart": "Starte das Beitreten von Raids neu"
+        },
+        "disableSpecific": "Deaktiviere automatisches Beitreten f�r bestimmte Raids"
+    },
+    "button": {
+        "ok": "Ok",
+        "cancel": "Schlie�en",
```
Comment (Idrinth) [inline src/languages/de.json]: Abbrechen

File: src/languages/de.json [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+        "tag": "Tag to screen-top",
+        "maxBoxes": "Es gibt keinen Platz f�r eine weitere Tierbox , bitte Schlie�en Sie eine."
+    },
+    "raids": {
+        "lastJoined": "Letzte beigetretene Raids",
+        "clear": "L�sche Raids",
+        "disableJoining": "Deaktiviere Beitreten der Raids ",
+        "clickCopy": "Klicke, um Raidlink zu kopieren",
+        "imports": {
+            "manually": "Importiere alle manuell",
```
Comment (Idrinth) [inline src/languages/de.json]: alle was?

File: src/languages/de.json [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+            "legend": "Legend",
+            "night": "Nightmare"
+        },
+        "tag": "Tag to screen-top",
+        "maxBoxes": "Es gibt keinen Platz f�r eine weitere Tierbox , bitte Schlie�en Sie eine."
+    },
+    "raids": {
+        "lastJoined": "Letzte beigetretene Raids",
+        "clear": "L�sche Raids",
+        "disableJoining": "Deaktiviere Beitreten der Raids ",
```
Comment (Idrinth) [inline src/languages/de.json]: sounds horrible, like a omputer voice being made to sound like a robot - would propose moving the deactivation to the end.

File: src/languages/de.json [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+            "normal": "Normal",
+            "hard": "Hard",
+            "legend": "Legend",
+            "night": "Nightmare"
+        },
+        "tag": "Tag to screen-top",
+        "maxBoxes": "Es gibt keinen Platz f�r eine weitere Tierbox , bitte Schlie�en Sie eine."
+    },
+    "raids": {
+        "lastJoined": "Letzte beigetretene Raids",
```
Comment (Idrinth) [inline src/languages/de.json]: Zuletzt...

File: src/languages/de.json [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+    "tier": {
+        "search": "Geben Sie den Namen des Bosses ein",
+        "diff": {
+            "normal": "Normal",
+            "hard": "Hard",
+            "legend": "Legend",
+            "night": "Nightmare"
+        },
+        "tag": "Tag to screen-top",
+        "maxBoxes": "Es gibt keinen Platz f�r eine weitere Tierbox , bitte Schlie�en Sie eine."
```
Comment (Idrinth) [inline src/languages/de.json]: Tierbox? Schließen lower case, Sie doesn't fit to the context, du would be better.

File: src/languages/de.json [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+    },
+    "tier": {
+        "search": "Geben Sie den Namen des Bosses ein",
+        "diff": {
+            "normal": "Normal",
+            "hard": "Hard",
+            "legend": "Legend",
+            "night": "Nightmare"
+        },
+        "tag": "Tag to screen-top",
```
Comment (Idrinth) [inline src/languages/de.json]: not translated

File: src/languages/de.json [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+    "default": "Unexpected error occurred. Please contact script developers (https://github.com/Idrinth/IDotD).",
+    "land": {
+        "lack": "Sie haben nicht genug Gold, um Geb�ude zu kaufen.",
+        "hour": " Gold pro Stunde jeweils",
+        "calc": "Berechne",
+        "avaible": "Verf�gbares Gold"
+    },
+    "tier": {
+        "search": "Geben Sie den Namen des Bosses ein",
+        "diff": {
```
Comment (Idrinth) [inline src/languages/de.json:48]: children not translated

File: src/languages/de.json [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+        "notification": {
+            "mention": "chat-mention notifications?",
+            "raid": "chat-raid notifications?",
+            "message": "chat-message notifications?"
+        }
+    },
+    "default": "Unexpected error occurred. Please contact script developers (https://github.com/Idrinth/IDotD).",
+    "land": {
+        "lack": "Sie haben nicht genug Gold, um Geb�ude zu kaufen.",
+        "hour": " Gold pro Stunde jeweils",
```
Comment (Idrinth) [inline src/languages/de.json]: jeweils -> und Gebäude

File: src/languages/de.json [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+            "login": "Login auf unerwartete Weise fehlgeschlagen",
+            "unknown": "Der eingegebene Benutzername ist dotd.idrinth.de nicht bekannt, m�chten Sie sich dort registrieren?"
+        },
+        "notification": {
+            "mention": "chat-mention notifications?",
+            "raid": "chat-raid notifications?",
+            "message": "chat-message notifications?"
+        }
+    },
+    "default": "Unexpected error occurred. Please contact script developers (https://github.com/Idrinth/IDotD).",
```
Comment (Idrinth) [inline src/languages/de.json]: not translated

File: src/languages/de.json [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+            "offline": "Sie sind nicht eingeloggt, bitte registrieren oder loggen Sie sich ein"
+        },
+        "error": {
+            "modify": "Konnte Benutzer nicht modifizieren",
+            "create": "Erstellung fehlgeschlagen",
+            "join": "Beitritt fehlgeschlagen",
+            "login": "Login auf unerwartete Weise fehlgeschlagen",
+            "unknown": "Der eingegebene Benutzername ist dotd.idrinth.de nicht bekannt, m�chten Sie sich dort registrieren?"
+        },
+        "notification": {
```
Comment (Idrinth) [inline src/languages/de.json:33]: not translated

File: src/languages/de.json [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+            "createAddChat": "Klicke, um neuen Chat zu erstellen",
+            "copyIdPasswort": "Kopiere Passwort&Id",
+            "deleteRoom": "Raum l�schen",
+            "leaveRoom": "Raum verlassen"
+        },
+        "texts": {
+            "settings": "Mehr Einstellungen bei ",
+            "account": "Account",
+            "title": "Chat",
+            "creditEmoticon": "Emoticons bereitgestellt durch ",
```
Comment (Idrinth) [inline src/languages/de.json]: bereitgestellt -> erschaffen, erstellt o.ä.

File: src/languages/de.json [hunk 17]
```diff
@@ -0,0 +1,101 @@
...
+            "invite": "Zum Chat einladen ",
+            "createChat": "Chat erstellen",
+            "joinChat": "Chat beitreten",
+            "createAddChat": "Klicke, um neuen Chat zu erstellen",
+            "copyIdPasswort": "Kopiere Passwort&Id",
+            "deleteRoom": "Raum l�schen",
+            "leaveRoom": "Raum verlassen"
+        },
+        "texts": {
+            "settings": "Mehr Einstellungen bei ",
```
Comment (Idrinth) [inline src/languages/de.json]: bei -> auf

File: src/languages/de.json [hunk 18]
```diff
@@ -0,0 +1,101 @@
...
+            "banUser": "Ban Benutzer",
+            "makeMod": "Promote zu Moderator",
+            "makeAdmin": "Promote zu Admin",
+            "makeUser": "Promote zu Benutzer",
+            "close": "Schlie�en",
+            "invite": "Zum Chat einladen ",
+            "createChat": "Chat erstellen",
+            "joinChat": "Chat beitreten",
+            "createAddChat": "Klicke, um neuen Chat zu erstellen",
+            "copyIdPasswort": "Kopiere Passwort&Id",
```
Comment (Idrinth) [inline src/languages/de.json]: copy to the end of the sentence

File: src/languages/de.json [hunk 19]
```diff
@@ -0,0 +1,101 @@
+{
+    "chat": {
+        "actions": {
+            "banUser": "Ban Benutzer",
+            "makeMod": "Promote zu Moderator",
+            "makeAdmin": "Promote zu Admin",
+            "makeUser": "Promote zu Benutzer",
+            "close": "Schlie�en",
+            "invite": "Zum Chat einladen ",
```
Comment (Idrinth) [inline src/languages/de.json]: Zu, not Zum

File: src/languages/de.json [hunk 20]
```diff
@@ -0,0 +1,101 @@
+{
+    "chat": {
+        "actions": {
+            "banUser": "Ban Benutzer",
```
Comment (Idrinth) [inline src/languages/de.json]: all the user ranks are not fully translated or not translated at all.

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: src/languages/de.json (hunk 20 above)
Comment (Ardenian) [reply src/languages/de.json]: Well, I am not sure how to translate 'ban', I'd expect gamers to know what it means. 'Verbanne Benutzer' does not sound very beautiful, does it. 
As for the other ones, Admin and Moderator should be appropriate, the term 'Promote' should be an equivalent to 'make', in any other case I would need some help on finding the correct term.

File: src/languages/de.json (hunk 18 above)
Comment (Ardenian) [reply src/languages/de.json]: Please rephrase, after my understanding it would be 'Passwort&Id kopiere'

File: src/languages/de.json (hunk 16 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed to 'erstellt'

File: src/languages/de.json (hunk 14 above)
Comment (Ardenian) [reply src/languages/de.json]: Fixed: 
"Ein unerwareteter Fehler ist eingetreten. Bitte kontaktiere den Entwickler auf (https://github.com/Idrinth/IDotD)."

File: src/languages/de.json (hunk 11 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed ( shouldn't call it fixed, right): 
"An den oberen Bildschirmrand anheften"

File: src/languages/de.json (hunk 3 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed it to that, though it is rather hard to understand for non-Coders, imo

File: src/languages/de.json (hunk 19 above)
Comment (Ardenian) [reply src/languages/de.json]: Fixed

File: src/languages/de.json (hunk 17 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed

File: src/languages/de.json (hunk 15 above)
Comment (Ardenian) [reply src/languages/de.json:33]: It is unclear to me how to translate this term. Something like "Benachrichtigungen für Chat-Nachrichten" ?

File: src/languages/de.json (hunk 13 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed

File: src/languages/de.json (hunk 12 above)
Comment (Ardenian) [reply src/languages/de.json:48]: I don't think translating the difficulty names does make sense, what would you say ? In the end, it does not offer any advantage, does it.

File: src/languages/de.json (hunk 9 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed

File: src/languages/de.json (hunk 8 above)
Comment (Ardenian) [reply src/languages/de.json]: changed

File: src/languages/de.json (hunk 7 above)
Comment (Ardenian) [reply src/languages/de.json]: Fixed and clarified

File: src/languages/de.json (hunk 6 above)
Comment (Ardenian) [reply src/languages/de.json]: Changed

File: src/languages/de.json (hunk 5 above)
Comment (Ardenian) [reply src/languages/de.json]: Sounds better, I agree, changed

File: src/languages/de.json (hunk 4 above)
Comment (Ardenian) [reply src/languages/de.json]: added

File: src/languages/de.json (hunk 2 above)
Comment (Ardenian) [reply src/languages/de.json]: fixed

File: src/languages/de.json (hunk 1 above)
Comment (Ardenian) [reply src/languages/de.json]: fixed

File: src/languages/de.json (hunk 14 above)
Comment (Idrinth) [reply src/languages/de.json]: die Entwickler (

File: src/languages/de.json (hunk 20 above)
Comment (Idrinth) [reply src/languages/de.json]: promote=befördern

Comment (Ardenian) [reply src/languages/de.json]: Isn't that exactly what happens when you 'make a user a moderator' ? I might get these make thingies on the wrong end

File: src/languages/de.json (hunk 14 above)
Comment (Ardenian) [reply src/languages/de.json]: changed

File: src/languages/de.json (hunk 18 above)
Comment (Idrinth) [reply src/languages/de.json]: obviously adjust it, "Passwort&Id kopieren"

File: src/languages/de.json (hunk 20 above)
Comment (Idrinth) [reply src/languages/de.json]: The point is you half-translated most of those things, "Promote zu Admin" is wierd, ban is usually germanized, so "Benutzer bannen" would make more sense.

Comment (Ardenian) [reply src/languages/de.json]: Aight, changed

**Review by Idrinth — APPROVED** [review]

**Comment by Idrinth** [comment]
LGTM

**Pull request merged**